Dockerfile

```diff
@@ -16,6 +16,7 @@ COPY main.go main.go
 COPY api/ api/
 COPY pkg/controllers/ pkg/controllers/
 COPY pkg/util/cert/ pkg/util/cert/
+COPY pkg/util/util  pkg/util/util
```

Comment:
Maybe you should just copy the code into the container so we don't get bit by this again..

Reply:
you mean copy the test code into the container (`pkg/util/testing`)? yeah I suppose that would be easier I just wasn't sure if it was good practice

Reply:
Kueue seems to do this: https://github.com/kubernetes-sigs/kueue/blob/main/Dockerfile#LL16C21-L16C21

I think if you want to not have the testing wrapper code in the container you could always move it to a separate module and explicity not include it.  Its probably not good practice to use a module called testing in the production code anyway.

Reply:
Maybe file it as a ticket?  Not sure you need to do it in this PR?

Reply:
Since Kueue does this I think it's probably fine, we can revisit later if we want